src/sass/Accordion.scss

```diff
@@ -135,3 +135,22 @@
     }
   }
 }
+
+@media screen and (min-width: 1004px) {
```

Comment:
it's for switch between one- and two-column accordions, there is no design for tablets yet, so I assumed that accordion column max width is `300px` x 2, plus margins and form width, this is how this breakpoint come.